modules/contribute/partials/component-terms.adoc

```diff
@@ -0,0 +1,78 @@
+The Stargate documentation repository uses a standard directory structure and nomenclature.
+Using link:https://antora.org[Antora] to build the docs requires this standard so that
+the static site generator (SSG) can locate the documentation components,
+collect the content files, and then build the site.
+
+Component::
+A documentation component contains AsciiDoc files, their optional assets and
+examples, one or more optional navigation files, and a component descriptor file
+per repository.
+All of the files in a component are versioned together.
++
+--
+While it is common to find all of a component's files in a single repository, a
+component can be distributed across several repositories.
+For instance, in the future, we may aggregate the Developer Guide component from
+_developers_guide_ and _code_examples_ repositories.
+
+A component isn't recognized by a repository or directory.
+Instead, the presence of a file named _antora.yaml_ indicates that the directories
```

Comment:
Do you typically italicize file names? I would've thought we would use `antora.yaml`.

Reply:
changed